sky/check.py

```diff
@@ -42,6 +42,7 @@ def check_one_cloud(
                                           ModuleType]]) -> None:
         cloud_repr, cloud = cloud_tuple
         assert capabilities is not None
+        cloud_capabilities = []
```

Comment:
This is not a List[CloudCapability], it's a list of tuples or whatever. That's fine, but can we have a type annotation and comment explaining for the format?